cli/ioctl/cmd/alias/aliasremove.go

```diff
@@ -19,12 +19,16 @@ import (
 
 // aliasRemoveCmd represents the alias remove command
 var aliasRemoveCmd = &cobra.Command{
-	Use:   "remove ALIAS",
+	Use:   "remove [ALIAS]",
```

Comment:
IMHO, alias operation doesn't need to be implicit

Reply:
ok